topiary-cli/Cargo.toml

```diff
@@ -77,7 +77,8 @@ contributed = [
 
 # Excluded by default
 experimental = [
-  "rust"
+  "rust",
+  "sdml"
```

Comment:
Ideally, `sdml` should come under the `contributed` feature, rather than `experimental`. However, if you feel it's still very much in an experimental state, then this is fine.

Reply:
I'll make the changes today, thanks!